cmd/extproc/mainlib/main.go

```diff
@@ -59,6 +63,12 @@ func parseAndValidateFlags(args []string) (extProcFlags, error) {
 		"log level for the external processor. One of 'debug', 'info', 'warn', or 'error'.",
 	)
 
+	fs.StringVar(&flags.promPort,
+		"promPort",
+		":9190",
+		"port for prometheus metrics, default is 9190",
```

Comment:
* Let's rename it to "addr" because it's a listen address, not just a port?
* Also, the flag library already includes the default in the message. If we add it to the description it will show up twice.
```suggestion
	fs.StringVar(&flags.promAddr,
		"promAddr",
		":9190",
		"address where Prometheus metrics are exposed",
```

Reply:
updated